### DISCUSSION TITLE
user online presence

### DISCUSSION THREAD

**Discussion by EdgarsLv** [supabase/supabase#5548] [Questions]
It is possible to listen on user online state like in firebase realtime database - onDisconnect()?  Not only when auth state changes because user can close browser tab without signOut. When user closes tab i want to update database, example: lastOnlineTime.

**Comment by nvsd** [comment]
I have the same [question](https://github.com/supabase/supabase/discussions/5544). Hopefully, this is possible.

**Comment by w3b6x9** [comment]
see https://github.com/supabase/supabase/discussions/5544#discussioncomment-2218497
